assets/js/components/KeyMetrics/MetricsSelectionPanel/Footer.js

```diff
@@ -228,21 +228,22 @@ export default function Footer( { savedMetrics } ) {
 
 	return (
 		<footer className="googlesitekit-km-selection-panel-footer">
-			{ saveError && (
-				<ErrorNotice
-					error={ saveError }
-					noPrefix={ selectedMetrics?.length < 2 }
-				/>
-			) }
 			<div className="googlesitekit-km-selection-panel-footer__content">
-				<p className="googlesitekit-km-selection-panel-footer__metric-count">
-					{ sprintf(
-						/* translators: 1: Number of selected metrics, 2: Number of selectable metrics */
-						__( '%1$d of %2$d selected', 'google-site-kit' ),
-						selectedMetrics?.length || 0,
-						4
-					) }
-				</p>
+				{ ! saveError ? (
+					<p className="googlesitekit-km-selection-panel-footer__metric-count">
+						{ sprintf(
+							/* translators: 1: Number of selected metrics, 2: Number of selectable metrics */
+							__( '%1$d of %2$d selected', 'google-site-kit' ),
+							selectedMetrics?.length || 0,
+							4
+						) }
+					</p>
+				) : (
+					<ErrorNotice
+						error={ saveError }
+						noPrefix={ selectedMetrics?.length < 2 }
+					/>
```

Comment:
It might be worth rendering a new `<ErrorNotice>` component here with the error for minimum selection requirement while rendering the API errors still at the original location.

The reason I propose this is because the API error length is unpredictable, and can cause a broken layout when it is too long. Example:

![image](https://github.com/google/site-kit-wp/assets/20284937/67f6a8ee-a48f-4481-8791-8ff604dbe8bd)

What do you think?

Reply:
@nfmohit Thanks, good catch! I included another error notice for API above the content